sgtypes/src/channel.rs

```diff
@@ -93,31 +93,17 @@ impl ChannelState {
 
 #[derive(Clone, Debug, Default)]
 pub struct WitnessData {
-    pub channel_sequence_number: u64,
```

Comment:
channel_sequence_number is needed to keep in WitnessData for submitting ChannelWriteSet transaction.